### PR TITLE
ras_ppcdiag: validate extract_opal_dump on PowerVM Platform

### PR DESCRIPTION
**Pull request by seeteena**
extract_opal_dump command is not supported on the PowerVM platform 
Add extract_opal_dump test to validate the support test.

### CODE REVIEW

File: ras/ras_ppcdiag.py [hunk 1]
```diff
@@ -114,6 +114,16 @@ def test_nvsetenv(self):
             self.fail("%s command(s) failed to execute  "
                       % self.fail_cmd)
 
+    @skipIf(IS_KVM_GUEST, "This test is not supported on KVM guest platform")
+    def test_extract_opal_dump(self):
+        """
+        extract_opal_dump test
+        """
+        self.log.info("======Executing extract_opal_dump tool test======")
+        if 'not supported' in process.run("extract_opal_dump -h").stderr.decode("utf-8"):
```
Comment (PraveenPenguin) [inline ras/ras_ppcdiag.py:123]: can we test other  supported option as well

Comment (seeteena) [reply ras/ras_ppcdiag.py:123]: @PraveenPenguin 

extract_opal_dump -h
extract_opal_dump help:
-A     - Don't acknowledge dump
-s dir - sysfs directory (default /sys)
-o dir - directory to save dumps (default /var/log/dump)
-m max - maximum number of dumps of a specific type to be saved
-w     - wait for a dump
-h     - help (this message)
------------------------------------------------
in order to check other options opaldump has to be present.

Comment (seeteena) [reply ras/ras_ppcdiag.py:123]: On supported PowerNV platform these command expects opaldump under /sys/firmware/opal/dump  directory. may be we can keep it as TODO in future like check for any dump file under this directory if present do the needed actions..  if not present validate all these command and return PASS. 

[root@ltc-wspoon14 ~]# extract_opal_dump -A
OPAL_DUMP[12572]: Error accessing sysfs: /sys/firmware/opal/dump (2: No such file or directory)
[root@ltc-wspoon14 ~]# extract_opal_dump -s /sys
OPAL_DUMP[12573]: Error accessing sysfs: /sys/firmware/opal/dump (2: No such file or directory)
[root@ltc-wspoon14 ~]#  extract_opal_dump -m 4
OPAL_DUMP[12574]: Error accessing sysfs: /sys/firmware/opal/dump (2: No such file or directory)
[root@ltc-wspoon14 ~]#  extract_opal_dump -w
OPAL_DUMP[12575]: Error accessing sysfs: /sys/firmware/opal/dump (2: No such file or directory)
[root@ltc-wspoon14 ~]#  extract_opal_dump -o /sys/
OPAL_DUMP[12576]: Error accessing sysfs: /sys/firmware/opal/dump (2: No such file or directory)
[root@ltc-wspoon14 ~]#

Comment (Naresh-ibm) [inline ras/ras_ppcdiag.py:123]: @seeteena also please take care of pycodestyle and pylint

**Review by Naresh-ibm — CHANGES_REQUESTED** [review]

**Comment by Naresh-ibm** [comment]
Also add the debug.log for both the scenarios where it is supported and where it is not supported

**Review by Naresh-ibm — APPROVED** [review]
LGTM

**Pull request merged**